packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -148,14 +136,25 @@ const TradersHub = () => {
                         ) : (
                             <ButtonToggleLoader />
                         )}
-                        {selected_platform_type === 'options' && <OptionsAndMultipliersListing />}
-                        {selected_platform_type === 'cfd' && <CFDsListing />}
+                        {renderOrderedPlatformSections(
+                            selected_platform_type === 'cfd',
+                            selected_platform_type === 'options'
+                        )}
                     </MobileWrapper>
                     <ModalManager />
                     {scrolled && <TourGuide />}
                 </div>
             </Div100vhContainer>
-            {is_eu_low_risk && <EUDisclamer />}
+            {is_eu_low_risk && (
+                <div className='disclamer'>
+                    <Text align='left' className='disclamer-text' size={isMobile() ? 'xxxs' : 'xs'}>
+                        <Localize
+                            i18n_default_text='<0>EU statutory disclaimer</0>: CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. <0>71% of retail investor accounts lose money when trading CFDs with this provider</0>. You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money.'
+                            components={[<strong key={0} />]}
+                        />
+                    </Text>
+                </div>
+            )}
```

Comment:
@amina-deriv defining a component inside component leads to recreation of component on each render, and also this change was suggested by ESLint plugin installed in VScode.
Since this component was only used once, i wrote it directly after conditional check.